x-pack/plugins/actions/server/sub_action_framework/sub_action_connector.ts

```diff
@@ -56,19 +57,7 @@ export abstract class SubActionConnector<Config, Secrets> {
   }
 
   private assertURL(url: string) {
```

Comment:
It looks like this method is used in a single place, which can be replaced by a call to the newly imported function - so I think we can remove this method.

Reply:
Can it be in a follow up? I had no intention of refactoring this part of the code, was following up with a comment @cnasikas left

Reply:
sure!